src/Hamiltonians/Molecular.jl

```diff
@@ -3,14 +3,20 @@
     
     MolecularMolecularHamiltonian(fd::ElemCo.FciDumps.FDump[, starting_address::FermiFS2C])
 
-Implements a Molecular Hamiltonian with [ElemCo.jl](https://elem.co.il). 
+Implements an electronic ab-initio Hamiltonian based on electron overlap integrals in
+FCIDUMP format. It can be used to describe the electronic structure of a molecule
+with fixed nuclear positions (i.e. under the Born-Oppenheimer approximation).
+The parse of FCIDUMP file depends on [ElemCo.jl](https://elem.co.il). 
 
 The expression of Hamiltonian under Bohn-Oppenheimer approximation is 
 ```math
 \\begin{aligned}
     \\hat{H}   & = \\hat{H}_0 + \\hat{H}_1 + \\hat{H}_2, \\\\
-    \\hat{H}_1 & = \\sum_{i,j} h_{ij} a^{\\dagger}_j a_i, \\\\
-    \\hat{H}_2 & = \\sum_{kl,ij} V_{kl,ij} a^{\\dagger}_{k} a^\\dagger_{l} a_j a_i.
+    \\hat{H}_1 & = \\sum_{\\sigma_i, \\sigma_j}\\sum_{i,j} h_{ij} 
+        a^{\\dagger}_{j,\\sigma_{j}} a_{i,\\sigma_{i}} \\delta_{ij}, \\\\
+    \\hat{H}_2 & = \\sum_{\\sigma_k, \\sigma_l} \\sum_{\\sigma_i, \\sigma_j} \\sum_{kl,ij} 
+        V_{kl,ij} a^{\\dagger}_{k,\\sigma_k} a^\\dagger_{l,\\sigma_l} a_{j,\\sigma_j} a_{i,\\sigma_i} 
+        - V_{kl,ji} a^{\\dagger}_{k,\\sigma_k} a^\\dagger_{l,\\sigma_l} a_{i,\\sigma_i} a_{j,\\sigma_j} \\delta_{ij}.
```

Comment:
Suggestion to simplify and make it more readable on the REPL by making use of unicode. The previous version was incorrect. Please double check that this is correct!
```suggestion
    Ĥ₁ & = \\sum_{i,j,σ} h_{ij}  a^†_{j,σ} a_{i,σ}, \\\\
    Ĥ₂ & = ½ \\sum_{i,j,σ,τ} V_{ijkl} a^†_{i,σ} a^†_{j,τ} a_{l,τ} a_{k,σ} .
```